app/main.py

```
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        print("Invalid command")
+        return
```

There are development/diagnostic `print()` calls: `print("Invalid command")` and `print("File not found")`. Clean Code checklist: "Don't forget to delete them when you are ready to commit and push your code." Remove these prints and let exceptions propagate or raise appropriate exceptions so tests can verify failures programmatically instead of relying on printed output.